demos/ragas-evaluation/rag-system-demo/Makefile

```
@@ -0,0 +1,75 @@
+.PHONY: deploy-all delete-all deploy-qwen3-14b-awq delete-qwen3-14b-awq \
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)